src/Article.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Box, Link, Text} from '@primer/components'
+import Octicon, {Note, Tag, LinkExternal, Megaphone, Broadcast} from '@githubprimer/octicons-react'
+
+export const iconForType = {
+  article: Note,
+  release: Tag,
+  podcast: Broadcast,
+  talk: Megaphone
+}
+
+const articleIcon = type => iconForType[type] || LinkExternal
+
+const articleDomain = url => {
+  return url.substring(url.indexOf('//') + 2, url.indexOf('/', 9))
+}
+
+const articleDate = date => {
+  const dateNames = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+  const day = new Date(date)
+  const now = new Date()
+  return `${dateNames[day.getMonth()]} ${day.getDate()}${
+    now.getFullYear() > day.getFullYear() ? `, ${day.getFullYear()}` : ''
+  }`
+}
+
+const Article = ({url, title, date, type}) => {
+  return (
+    <Box mb={7}>
+      <Link href={url}>
+        <Text fontFamily="mono" fontSize={3}>
+          {title}
+        </Text>
+      </Link>
+      <Box mb={3}>
+        <Text f={1} color="blue.1">
```

Comment:
IMO this should be 14px to match the label type underneath it, not sure what f={1} outputs but looks like 16px right now